Jenkinsfile

```diff
@@ -6,7 +6,7 @@ pipeline {
         sh '''
 mkdir -p build
 cd build
-cmake .. -DBUILD_TESTS=ON -DUSE_CUDNN=ON -DUSE_SIMSEARCH=ON -DUSE_TSNE=ON -DUSE_XGBOOST=ON -DUSE_CAFFE2=ON -DCUDA_ARCH="-gencode arch=compute_61,code=sm_61"
+cmake .. -DBUILD_TESTS=ON -DUSE_CUDNN=ON -DUSE_SIMSEARCH=ON -DUSE_TSNE=ON -DUSE_XGBOOST=ON -DUSE_CAFFE2=ON -DUSE_NCNN -DCUDA_ARCH="-gencode arch=compute_61,code=sm_61"
```

Comment:
It should be `-DUSE_NCNN=ON`. Please fix it, and see whether tests are passing.

Reply:
Yes, fixed.